src/model/Clearing.java

```diff
@@ -3,19 +3,23 @@
 import java.util.Arrays;
 
 public class Clearing {
-	
+
     public String type;
     public Garrison dwelling = null; 			// not sure what type this should actually be
     public int location;		 		// indicating which clearing on the tile this is CHANGED BACK TO AN INT
     public Clearing[] connections;
     public int nextConnection =0;
     public Tile parent;
-    
+    public Treasure[] treasures;
+
+
     Clearing(int location, Tile parent) {
         this.type     = "";
         this.location = location;
-        this.parent = parent;
-        connections = new Clearing[4];		//4 is the most connections had by any 
+        this.parent   = parent;
+        connections   = new Clearing[4];		//4 is the most connections had by any
+        treasures     = new Treasure[10]; // arbitrary number
+        // TODO: how should the treasures be added to the tile?
```

Comment:
@SunsetBlade I wasn't sure if the board would place the treasures around when its setup, or what